src/components/map/Map.tsx

```diff
@@ -209,7 +211,8 @@ export const Map = ({
           cameraRef.current.fitBounds(
             [minLng - deltaLng, minLat - deltaLat],
             [maxLng + deltaLng, maxLat + deltaLat],
-            0
+            0,
+            1000
```

Comment:
what is the fourth value here that is set with 1000 now?

Reply:
![image](https://github.com/user-attachments/assets/be2a56b6-37de-4e00-a85e-ef48a8a38d17)

Reply:
I added this to the commit.

- added value 1000 to the fitBounds function to show the cluster in an animated way

Reply:
i know. but this does not clearly say what the value stands for exactly